### PR TITLE
Adding support for the subtitle streams.

### PR DESCRIPTION
**Pull request by AwkwardSandwich**

### CODE REVIEW

**Comment by beanieboi** [comment]
Could you please add a sample file and test the subtitles in the parser_spec.rb

**Comment by AwkwardSandwich** [comment]
Hello, i think everything is in order!  I noticed the 'sample video.m4v' already had a subtitle stream.  I could write the tests using that file.  Thanks!

**Comment by AwkwardSandwich** [comment]
I am also wondering how we could go about changing the MAX_VERSION to 2.5.2 (current version of ffmpeg).  Is there any test that could to be written?

File: lib/ffprober/subtitle_stream.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+module Ffprober
+  class SubtitleStream < Stream
+  end
+end
```
Comment (beanieboi) [inline lib/ffprober/subtitle_stream.rb]: can you please add a newline at the end

**Comment by beanieboi** [comment]
can you please add an entry to `Changes.md`

**Comment by AwkwardSandwich** [comment]
Absolutely, would that go under 0.4.2?  thank you.

**Comment by beanieboi** [comment]
just name it `master`, i'll update the version on release

**Pull request merged**